man/tpm2tss-genkey.1.md

```diff
@@ -32,6 +32,12 @@ key information. This file can then be loaded with OpenSSL using
   * `-c <curve>`, `--curve <curve>`:
     If alg ecdsa is chosen, the curve for ecc (default: nist_p256)
 
+  * `-i <file>`, `--importpub <file>`:
+    Public key (TPM2B_PUBLIC) to be imported. Requires `-k`.
+
+  * `-k <file>`, `--importtpm <file>`:
+    The (encrypted) private key (TPM2B_PRIVATE) to be imported.
```

Comment:
doesn't `-k` need `-i` as well?

Change `-i` to `-u` and `-k` to `-r` to be consistent with tpm2-tools.